src/services/address_service.js

```diff
@@ -2,19 +2,19 @@ import baseService from './base_service';
 
 export default (easypostClient) =>
   class AddressService extends baseService(easypostClient) {
-    static _name = 'Address';
+    static #_name = 'Address';
```

Comment:
We don't want these to be both pounded (#) and underscored. They should only be one or the other I'd think

Reply:
Oops, this is a typo here, but for the private fields it should be using `#`